layouts/partials/breadcrumbs.html

```diff
@@ -0,0 +1,26 @@
+{{ $url := replace .Permalink ( printf "%s" .Site.BaseURL) "" }}
+{{ $.Scratch.Add "path" .Site.BaseURL }}
+
+{{ $.Scratch.Add "breadcrumb" (slice (dict "url" .Site.BaseURL "name" "home" "position" 1 )) }}
+{{ range $index, $element := split $url "/" }}
+{{ $.Scratch.Add "path" $element }}
```

Comment:
We could add some indentation here to help with readability.
```suggestion
  {{ $.Scratch.Add "path" $element }}
```